frontend/src/metabase/core/components/TabButton/TabButton.tsx

```diff
@@ -22,14 +22,16 @@ import {
   TabContext,
   TabContextType,
 } from "../Tab";
+import { TabButtonMenu } from "./TabButtonMenu";
 import {
   TabButtonInput,
   TabButtonRoot,
   MenuButton,
   TabButtonInputWrapper,
   TabButtonInputResizer,
 } from "./TabButton.styled";
-import TabButtonMenu from "./TabButtonMenu";
+
+export const INPUT_WRAPPER_TEST_ID = "tab-button-input-wrapper";
```

Comment:
Normally we prefer using other methods of selecting an element in tests, for example aria-labels, but in this case we don't want to add an aria-label since the component already has aria-label and aria-labeled by props, and this specific wrapper element isn't relevant to the user at all, it is just there to make the resizing hack work and to attached the double click event listener to.